plugins/algoltek-usb/fu-algoltek-usb-common.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2023 Ling.Chen <ling.chen@algoltek.com.tw>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <glib-object.h>
+
+#define ALGOLTEK_DEVICE_USB_TIMEOUT 3000 /* ms */
+/* command types in USB messages from PC to device */
+#define ALGOLTEK_USB_REQUEST_CMD01 0x08
```

Comment:
Can we have more descriptive names?

Reply:
I saw the plugin used obfuscated name like the file in  [tps6598x](https://github.com/Algoltek/fwupd/blob/main/plugins/ti-tps6598x/fu-ti-tps6598x-common.h).

Reply:
If you can avoid it it's easier to maintain in the future if not obfuscated.

Think about it like 10 years from now if a new guy picked up this plugin and needed to add support for a device with a slight variation on your update protocol. With function names like this and commands like this they'll have a really hard time finding that nuance.

Reply:
> the plugin used obfuscated name like the file

Those aren't obfuscated :) It's literally what the datasheet says. I agree with @superm1 -- it's easy to guess what each thing is (we've seen the same kind of protocol from other vendors) and the obfuscation is pretty useless and makes it very hard to review and then debug in the field.

Reply:
Thanks for the suggestion. Before LVFS, Algoltek already provided our partners with the standalone software which is named FUU and running on Microsoft Windows, Apple macOS, and Linux seperately for updating the firmware of Algoltek ASIC through USB interface (or DisplayPort AUX interface). The customized USB vendor commands all follow the updating procedure of Algoltek In-System Programming(ISP) mechanism. The plugin or even FUU cannot be altered without modifying the Algoltek In-System Programming(ISP) mechanism.

Reply:
Ohh we don't need anything modified, we just need proper enums. E.g. remove CMD01 in the source code and use SET_ADDRESS or whatever it actually is.

Reply:
Thank you for your advice. I've modified the obfuscated words in the source code.